onnxruntime/wasm/api.cc

```diff
@@ -246,6 +246,12 @@ int OrtGetTensorData(OrtValue* tensor, int* data_type, void** data, size_t** dim
   size_t* p_dims = nullptr;
   void* p_string_data = nullptr;
 
+  ONNXType* TensorType;
```

Comment:
use camelCase for local variable.

Reply:
!IMPORTANT! 

you should not declare a pointer here on a stack. should declare a variable of type ONNXType ( which is enum ). Otherwise the call to GetValueType() below will write to unexpected memory